src/main/java/ladder/view/ResultView.java

```diff
@@ -0,0 +1,65 @@
+package ladder.view;
+
+import ladder.domain.game.Ladder;
+import ladder.domain.game.Line;
+import ladder.domain.game.Name;
+import ladder.domain.game.Names;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+import static java.util.stream.Collectors.joining;
+
+/**
+ * Created By mand2 on 2020-12-07.
+ */
+public class ResultView {
+
+    public static final String RESULT = "실행결과\n";
```

Comment:
OS 마다 줄바꿈을 인식하는게 다르기 때문에 jdk 1.7부터 도입된 현재 OS에 맞는 줄바꿈을 얻어오도록 하면 좋을 것같아요!
```suggestion
    public static final String RESULT = "실행결과" + System.lineSeperator();
```